src/main/java/nextstep/subway/applicaion/dto/FavoriteResponse.java

```diff
@@ -0,0 +1,46 @@
+package nextstep.subway.applicaion.dto;
+
+import nextstep.subway.domain.Favorite;
+
+import java.util.List;
+
+public class FavoriteResponse {
+
+    private Long id;
+    private StationResponse source;
+    private StationResponse target;
+
+    public FavoriteResponse(Long id, StationResponse source, StationResponse target) {
+        this.id = id;
+        this.source = source;
+        this.target = target;
+    }
+
+    public static FavoriteResponse of(Favorite favorite) {
+        return new FavoriteResponse(favorite.getId(),StationResponse.of(favorite.getSource()),StationResponse.of(favorite.getTarget()));
```

Comment:
```suggestion
                return new FavoriteResponse(
                favorite.getId(),
                StationResponse.of(favorite.getSource()),
                StationResponse.of(favorite.getTarget())
        );
```

코드를 한 줄에 다 쓰는 것보다 적절한 엔터를 넣어주는게 코드 읽기에 좋습니다.